docs/admin/whats-new.md

```diff
@@ -0,0 +1,163 @@
+- **Self-Service Admin Transfer**: Easily reassign space owners and assign multiple owners for shared responsibility.
+- **Easy Permission Control**: Manage user access based on the roles Viewers, Editors, Managers.
+
+---
+
+## **📝 Real-Time Collaboration – Web Office (Collabora)**
+
+- **Real-Time Editing**: Multiple users can work on documents at the same time.
+- **No File Transfers**: Edit directly in the browser always on the latest version.
+- **No Software Installation**: Works entirely online, reducing costs and complexity.
+
+---
+
+## **🔎 Smart Search & Organization**
+
+- **Full-Text Search**: Find files quickly, even in scanned documents.
+- **Live Search**: Instant results as you type.
+- **Image-to-Text Recognition**: Extract text from images and scanned files (OCR - Optical Character Recognition).
+- **Advanced Filters**: Search by document type, tags, date, or storage location.
+
+### **Tags**
+
+- **Flexible Organization**: Categorize files beyond folder structures.
+- **Quick Search**: Find files faster using keywords.
+- **Audit & Compliance**: Ensure traceability of important documents.
+
+---
+
+## **📜 File Versioning & Recovery**
+
+- **Easy Restoration**: Roll back to previous file versions.
+- **Accidental Change Protection**: Recover files in case of unintended edits.
+- **Deleted File Recovery**: Restore files from the trash bin.
+
+---
+
+## **⚡ High-Performance, User-Friendly Interface**
+
+- **Familiar Desktop Features**:
+  - **Right-click** context menus.
+  - **Drag & Drop** support between web and desktop.
+  - **Keyboard Shortcuts**: (Ctrl + C, Ctrl + X, Ctrl + V).
+  - **Multi-Selection**: (Ctrl + Click, Shift + Click).
+- **Instant UI Updates**: No need to refresh when a file is modified.
+
+---
+
+## **🛡️ Security & Compliance**
+
+### **File Firewall**
+
+- **Block Malicious Uploads**: Prevent unwanted file types like **.exe or macro files**.
+- **Allowlist & Denylist Support**: Define permitted and blocked file types.
+
+### **ICAP Antivirus Interface**
+
+- **Automated Malware Scanning**: Scan all uploaded files before they reach user devices.
+- **Early Threat Detection**: Prevent infections before they spread.
+- **Compliance Assurance**: Meets security standards for home and enterprise storage.
+
+### **Zero-Knowledge Principle**
+
+- **Admin Privacy Controls**: Admins manage spaces without accessing file content.
+- **User-Owned Data**: Only designated users can access stored files.
+
+### **Access Control**
+
+- **Granular Permission Management**: Role-Based Access Control.
+
+### **GDPR & Accessibility Compliance**
+
+- **GDPR Compliant**: Fully meets **EU data protection regulations**.
+- **BITV 2.0 / WCAG 2.1 Compliance**: Ensures **accessible** web usage for all users.
+
+---
+
+## **📦 Unlimited File Size**
+
+- **No File Size Limits**: Upload large files without restrictions.
+
+---
+
+## **🛠️ Web Extension System**
```

Comment:
should we mark as `soon`?

Reply:
@kulmann `soon` or `there` - what would you say?

Reply:
It's already there